src/scss/App.scss

```diff
@@ -158,3 +158,114 @@ a.event-button:focus {
   background-color: white;
   cursor: not-allowed;
 }
+
+// Event Filtering
+
+.filter-bar__browse--menu-wrapper {
+  position: relative;
+}
+
+@media (min-width:851px) {
+  .filter-bar__browse--menu-wrapper-open .filter-bar__browse--dropdown {
+    display: block;
+  }
+}
+
+.filter-bar__browse--menu-wrapper .filter-bar__browse--menu::after {
+  right: calc(1.1rem + 1px);
+  bottom: calc(-0.25rem - 1px);
+  background-color: #fff;
+  z-index: 100;
+}
+
+.filter-bar__browse--menu-wrapper .filter-bar__browse--menu::before {
+  right: 1.1rem;
+  bottom: calc(-0.25rem - 2px);
+  background-color: #cbd6e2;
+  border: 1px solid #cbd6e2;
+}
+
+.filter-bar__browse--menu-wrapper .filter-bar__browse--menu::before,
+.filter-bar__browse--menu-wrapper .filter-bar__browse--menu::after {
+  display: inline-block;
+  position: absolute;
+  border-radius: .1875rem 0 0;
+  -webkit-transform: rotate(45deg);
+  transform: rotate(45deg);
+  width: 1rem;
+  height: 1rem;
+}
+
+@media (min-width:851px) {
+  .filter-bar__browse--menu-wrapper-open .filter-bar__browse--menu::before,
+  .filter-bar__browse--menu-wrapper-open .filter-bar__browse--menu::after {
+    content: '';
+  }
+}
+
+.filter-bar__browse--menu {
+  position: relative;
+}
+
+@media (min-width:851px) {
+  .filter-bar__browse--title-wrapper {
+    display: inline-block;
+    padding: 0.6rem 0;
+  }
+}
+
+.filter-bar__browse--title {
+  vertical-align: 0.5em;
+}
+
+.filter-bar__browse--dropdown {
+  cursor: auto;
+  font-size: 0.875rem;
+  margin: 0 auto;
+  padding: 0 1rem 0 1rem;
+  position: absolute;
+  background-color: #fff;
+  z-index: 95;
+  left: 3em;
+  top: 3.55em;
+}
+
+@media (min-width:768px) {
+  .filter-bar__browse--dropdown {
+    border: 1px solid #cbd6e2;
+    border-radius: .1875rem;
+    box-shadow: 0 1px 24px rgba(45,62,80,0.12);
+  }
+}
+
+@media (min-width:851px) {
+  .filter-bar__browse--dropdown {
+    display: none;
+  }
+}
+
+input[type="checkbox"] {
```

Comment:
We don't want to include any styles that style HTML elements directly because CSS for the app could negatively impact the display of the elements used elsewhere on the page. Instead I suggest adding classnames to the elements and putting the code in `EventFilterCheckbox`.